packages/background/src/controllers/ExchangeRatesController.ts

```diff
@@ -135,19 +647,18 @@ export class ExchangeRatesController extends BaseController<ExchangeRatesControl
         const { symbol } = this.networkNativeCurrency;
         // Rates format: {[tokenTicker]: [price: number]}
         const rates = this.store.getState().exchangeRates;
-        const currencyApiId =
-            RATES_IDS_LIST[
-                this.networkNativeCurrency.symbol as keyof typeof RATES_IDS_LIST
-            ];
 
         // Get network native currency rate
-        const nativeCurrencyRates = (
-            await this._getNetworkNativeCurrencyRate()
-        )[currencyApiId];
+        const nativeCurrency =
+            this._preferencesController.nativeCurrency.toLowerCase();
 
-        // Response -> { ethereum : { [nativeCurrency] : rate }}
-        rates[symbol] =
-            nativeCurrencyRates[this._preferencesController.nativeCurrency];
+        rates[symbol] = await this._exchangeRateService.getRate(
```

Comment:
What happens if the `chainLinkService` returns 0? Shouldn't we try the `coingeko` implementation.